### ISSUE TITLE
Question about gated deconv

### ISSUE THREAD

**Issue by r06922019** [run-youngjoo/SC-FEGAN#37]
Hi,

Thanks for your great work.
In your code, 
https://github.com/run-youngjoo/SC-FEGAN/blob/master/ops.py#L50
this line states `g = tf.nn.sigmoid(deconv)`
However, IMHO the sigmoid should be applied to g itself to construct the gate
It should be `g = tf.nn.sigmoid(g)` right?
Did I misunderstand anything?

Thanks for your great work again!

**Comment by r06922019** [comment]
Also, according to this line,
https://github.com/run-youngjoo/SC-FEGAN/blob/master/ops.py#L46
`g = tf.nn.conv2d_transpose(input_, w, output_shape=output_shape,
                    strides=[1, d_h, d_w, 1])`
it seems that the gate `g` shares the same `w` used in `deconv`
Can you provide some thoughts on this?
Thanks again

**Comment by abtExp** [comment]
@r06922019 you're right.

**Comment by abtExp** [comment]
You're right.
